library/catalog/urls.py

```diff
@@ -6,7 +6,7 @@
 urlpatterns = [
     path('home/', views.home, name='library-home'),
     path('registration/', views.registration, name='library-registration'),
-    path('registration_unique_validation/', views.unique_registration_check,
+    path('registration_validation/<str:field_value>', views.unique_registration_check,
```

Comment:
Turn back to `registration_unique_validation/`

Reply:
> Change back to `registration_unique_validation/`

done